packages/google_maps_flutter/lib/src/google_maps_controller.dart

```diff
@@ -351,11 +350,28 @@ class GoogleMapController {
         console.log('trafficLayer detached!!');
       }
     ''';
-    await mapView.evaluateJavascript(command);
+    await (await controller).evaluateJavascript(command);
+  }
+
+  Future<void> _setMoveCamera(String options) async {
+    await _callMethod(await controller, 'moveCamera', [options]);
+  }
+
+  Future<void> _setPanTo(String options) async {
+    await _callMethod(await controller, 'panTo', [options]);
+  }
+
+  Future<void> _setPanBy(String options) async {
+    await _callMethod(await controller, 'panBy', [options]);
+  }
+
+  Future<void> _setFitBounds(String options) async {
+    await _callMethod(await controller, 'fitBounds', [options]);
   }
 
   Future<String> _callMethod(
       WebViewController mapView, String method, List<Object?> args) async {
+    print('callMethod: $method($args)');
```

Comment:
How about removing these logs? (I mean ``print`` ) 
Even if it is necessary during development, I would like to remove it in the end. I couldn't find such a log in other plugins.

Reply:
You are right. It will be removed before it is merged with master branch.